### PR TITLE
Initial commit

### PR DESCRIPTION
**Pull request by XDIJK**

### CODE REVIEW

File: 4817_Sprint/04 - xander/src/com/company/Bogosort.java [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+      int temp = array[index];
+      array[index] = array[i];
+      array[i] = temp;
+    }
+    return array;
+  }
+
+  private static boolean isSorted(int[] unsortedArray) {
+    for(int i = 0; i < unsortedArray.length; i++) {
+      if(i == unsortedArray.length-1) {
```
Comment (adiazQualogy) [inline 4817_Sprint/04 - xander/src/com/company/Bogosort.java]: blank space after the if

File: 4817_Sprint/04 - xander/src/com/company/Bogosort.java [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+        return true;
+      } else if( unsortedArray[i] > unsortedArray[i+1] ) {
+        break;
+      }
+    }
+    return false;
+  }
+
+  public static int[] sort(int[] unsortedArray) {
+    while(!isSorted(unsortedArray)) {
```
Comment (adiazQualogy) [inline 4817_Sprint/04 - xander/src/com/company/Bogosort.java]: blank space after the while

File: 4817_Sprint/04 - xander/src/com/company/Main.java [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+        } catch (InterruptedException | ExecutionException e) {
+            e.printStackTrace();
+        }finally {
+            if(service != null) service.shutdown();
+        }
+    }
+
+    static Callable<int[]> initizalizeBogosort() {
+        return () -> {
+            int[] unsorted = new int[11];
```
Comment (adiazQualogy) [inline 4817_Sprint/04 - xander/src/com/company/Main.java]: avoid usage of magical numbers

File: 4817_Sprint/04 - xander/src/com/company/Bogosort.java [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+      if(i == unsortedArray.length-1) {
+        return true;
+      } else if( unsortedArray[i] > unsortedArray[i+1] ) {
+        break;
+      }
+    }
+    return false;
+  }
+
+  public static int[] sort(int[] unsortedArray) {
```
Comment (adiazQualogy) [inline 4817_Sprint/04 - xander/src/com/company/Bogosort.java]: this can be private

File: 4817_Sprint/04 - xander/src/com/company/Main.java [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+           int[] firstSortedArray = service.invokeAny(callables);
+           service.shutdownNow();
+
+           end = (System.currentTimeMillis() - start)/1000;
+           System.out.println("Parallel first result" + " " + Arrays.toString(firstSortedArray) + end);
+
+        } catch (InterruptedException | ExecutionException e) {
+            e.printStackTrace();
+        }finally {
+            if(service != null) service.shutdown();
```
Comment (adiazQualogy) [inline 4817_Sprint/04 - xander/src/com/company/Main.java]: use brackets for the if please

File: 4817_Sprint/04 - xander/src/com/company/Bogosort.java (hunk 1 above)
Comment (ssoegriemsingh) [reply 4817_Sprint/04 - xander/src/com/company/Bogosort.java]: Also at the for?

File: 4817_Sprint/04 - xander/src/com/company/Main.java (hunk 5 above)
Comment (ssoegriemsingh) [reply 4817_Sprint/04 - xander/src/com/company/Main.java]: space before the finally

**Pull request merged**